ldes-fragmentisers/ldes-fragmentisers-substring/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/substring/constants/SubstringConstants.java

```diff
@@ -7,6 +7,7 @@ public class SubstringConstants {
 	private SubstringConstants() {
 	}
 
+	public static final String ROOT_SUBSTRING = "";
```

Comment:
Actually, I think it's better to include these static variables where they are needed than bundle them in a constants class (I know I probably created it in the first place)